src/components/hoc/InputText.js

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import styled from "styled-components";
+import { theme } from "../../theme/theme";
+
+export default function InputText({ value, onChange, Icon, ...others }) {
+  // state (état, données)
+
+  // comportement (fonctions)
+
+  // affichage (JSX)
+  return (
+    <InputTextStyled>
+      {Icon && Icon}
+      <input onChange={onChange} value={value} type="text" {...others} />
+    </InputTextStyled>
+  );
+}
+const InputTextStyled = styled.div`
```

Comment:
```suggestion
const TextInputStyled = styled.div`
```

J'ai eu un oubli dans la vidéo, ça aurait bien du être TextInputStyled ici.

Reply:
J'ai bien fait ou pas, je ne comprends pas?!

Reply:
Résolu